README.md

```diff
@@ -59,7 +60,7 @@ We expect attendees to have at least the following level of experience:
 - R
   - **Mandatory**: Install R and RStudio ([_link_](https://github.com/bcgov/bcgov-data-science-resources/wiki/Installing-R-&-RStudio))
   - **Mandatory**: Install R packages (more to come): <br>
-      `c("tidyverse", "sf", "sp","raster",`<br>
+      `install.packages("tidyverse", "sf", "sp","raster",`<br>
```

Comment:
ln 63 should be: `install.packages(c("tidyverse", "sf", "sp","raster",`<br>
ln 65 needs extra closing brace